src/main/java/io/jenkins/pluginhealth/scoring/scores/PluginMaintenanceScoring.java

```diff
@@ -46,25 +47,34 @@ protected ScoreResult doApply(Plugin plugin) {
         final ProbeResult dependabotProbeResult = plugin.getDetails().get(DependabotProbe.KEY);
         final ProbeResult dependabotPullRequestResult = plugin.getDetails().get(DependabotPullRequestProbe.KEY);
         final ProbeResult cdProbeResult = plugin.getDetails().get(ContinuousDeliveryProbe.KEY);
+        final ProbeResult contributingGuidelinesProbeResult = plugin.getDetails().get(ContributingGuidelinesProbe.KEY);
 
-        if (jenkinsfileProbeResult == null || jenkinsfileProbeResult.status().equals(ResultStatus.FAILURE)) {
-            return new ScoreResult(KEY, 0, COEFFICIENT);
+        float score = 0.0f;
+
+        if (jenkinsfileProbeResult != null && jenkinsfileProbeResult.status().equals(ResultStatus.SUCCESS)) {
+            score += 0.65f;
+        }
+
+        if (contributingGuidelinesProbeResult != null && contributingGuidelinesProbeResult.status().equals(ResultStatus.SUCCESS)) {
+            score += 0.15f;
         }
 
-        if (dependabotProbeResult == null || dependabotProbeResult.status().equals(ResultStatus.FAILURE)) {
-            return new ScoreResult(KEY, .75f, COEFFICIENT);
+        if (dependabotProbeResult != null && dependabotProbeResult.status().equals(ResultStatus.SUCCESS)) {
+            score += 0.15f;
         }
 
         if (dependabotPullRequestResult != null && dependabotPullRequestResult.status().equals(ResultStatus.SUCCESS)
             && Integer.parseInt(dependabotPullRequestResult.message()) > 0) {
-            return new ScoreResult(KEY, .75f, COEFFICIENT);
+            score -= 0.15f;
         }
 
-        if (cdProbeResult == null || cdProbeResult.status().equals(ResultStatus.FAILURE)) {
-            return new ScoreResult(KEY, .9f, COEFFICIENT);
+        if (cdProbeResult != null && cdProbeResult.status().equals(ResultStatus.SUCCESS)) {
+            score += 0.05f;
         }
 
-        return new ScoreResult(KEY, 1, COEFFICIENT);
+        score = Math.round(score * 100.0) / 100.0f; // Prevent rounding errors
```

Comment:
Can you explain why you need this?

Reply:
Taking an example, `0.65f + 0.15f + 0.05f` was giving a result `0.84999996f` due to the way that these numbers are represented in binary. It was caught by the unit tests.

I am using `Math.round` method to round the result to two decimal places.

Reply:
Alright, this is clear enough for me. Thanks.